### DISCUSSION TITLE
Problem installing micropython/lora on a STM32F411

### DISCUSSION THREAD

**Discussion by davefes** [micropython/micropython#12959] [STM32 / Pyboard] [answered]
Using `mpremote mip install lora-sync` and `mpremote mip install lora-sx127x` they appear at /flash/lib/lora on the device.

In `simple_rxtx.py` I had to add `import lib` before the line ` from lora import SX1276X` so looks like I didn't understand the install process correctly.  Or is there an omission in the example?

I can get this at both units:
```py
Sending...
Receiving...
Timeout!
```
but no `Hello world from MicroPython`.

**Comment by glenn20** [comment]
Re. the import pronlem, I've no experience with the stm controllers, but you might want to check that `/flash/lib` is in the default `sys.path`. If not, you should add it on startup.

**Reply by davefes** [reply]
Thanks for the hint:

>>> sys.path
[' ', '.frozen', '/flash', '/flash/lib']

lora is a directory under lib

**Reply by jimmo** [reply, marked as answer]
`/flash/lib` is in the default sys.path on stm32.

@davefes In the `simple_rxtx.py` example, [the line is](https://github.com/micropython/micropython-lib/blob/master/micropython/lora/examples/simple_rxtx/simple_rxtx.py#L8) `from lora import SX1276`... you've added an extra `X` to the end.

**Reply by davefes** [reply]
Took a minute to work that out, I actually had:
```py
def get_modem():
    import lib
    from lora import SX1276
```
but appears that when I started this journey I was throwing Xs around.

Thanks for the confirming sys.path.
